vm/src/vm/vm_new.rs

```diff
@@ -270,7 +270,11 @@ impl VirtualMachine {
         } else {
             Vec::new()
         };
-        self.new_exception(self.ctx.exceptions.stop_iteration.to_owned(), args)
+        let exc = self.new_exception(self.ctx.exceptions.stop_iteration.to_owned(), args.clone());
+        exc.as_object()
```

Comment:
I think this is conceptionally calling __init__. I like this approach for performance, but please leave a comment this is intending calling init.
Actually, creating a shared function and calling it would be perfect

Reply:
Thank you for comments!
I created a `set_exception_attr()` function and made `new_stop_iteration` to call it to initialize `value` attribute.